nautobot/core/api/metadata.py

```diff
@@ -144,16 +192,25 @@ def determine_actions(self, request, view):
                 view.request = request
         return actions
 
-    def determine_view_options(self, request, view, serializer):
+    def get_list_display_fields(self, serializer):
+        """Try to get the list display fields or default to an empty list."""
+        serializer_meta = getattr(serializer, "Meta", None)
+        return getattr(serializer_meta, "list_display_fields", None) or []
```

Comment:
Not sure name-wise about `list_display_fields`, would something like `default_list_display_fields` be clearer as to its meaning?

Reply:
We could just call them `default_columns` like they are in the tables?